.github/workflows/test.yml

```
@@ -30,6 +30,11 @@ jobs:
           requirements_file: requirements.yml
 
   tests:
+    strategy:
```

you probably also want:
```suggestion
    strategy:
      fail-fast: false
```
otherwise it will cancel "the other" branch when one fails, which in most case is not desired?